File: week1_basic13/12.js
// Return the given array, after setting any negative values to zero

function negZero(arr) {
    for (let i=0; i < arr.length; i++) {
        if (arr[i] < 0) {
            arr[i] = 0
        }
    }
    return arr
}

console.log(negZero([2,-2,-4,2,5,-10,12]))

// es6

const negZeroEs6 = arrEs6 => {
    arrEs6 = arrEs6.map(num => num < 0 ? 0 : num)
    return arrEs6
}

console.log(negZeroEs6([2,-2,-4,2,5,-10,12]))

File: interview_prep/week1_basic13/12.py
# Return the given array, after setting any negative values to zero

def negZero(arr):
    for i in range(0, len(arr)-1, 1):
        if arr[i] < 0:
            arr[i] = 0
    return arr

print(negZero([2,-2,-4,2,5,-10,12]))

